Assets/MixedRealityToolkit-SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -190,10 +198,37 @@ protected virtual void AttachToNewTrackedObject()
                 case TrackedObjectType.Head:
                     TrackTransform(CameraCache.Main.transform);
                     break;
-                // Other cases will come online as ControllerFinder is ported appropriately.
+                case TrackedObjectType.LeftController:
+                    var leftController = GetVisualizerForHand(Handedness.Left);
```

Comment:
Eric, I think you nailed it right on the head. I'm down to update the terminology to better reflect mvvm.

Reply:
Agree, an interesting way to re"visualize" the concept to another pattern.
* A Device is the "Model" - data
* The DeviceManager/MRManager are the View - connecting a model to a vm
* The scene "things" are the view models - providing cuntionality

Reply:
Hmmm, maybe I don't understand the role of the DeviceManager/MRManager, @SimonDarksideJ ?

>  the View - connecting a model to a vm

In MVVM, a View doesn't connect a model to a VM, in fact, it may not necessarily even correspond to a specific model of the data (though it does correspond to the *kind* of data that's stored in that model [a birthdate View can't connect to a Model of a audio clip]). The ViewModel is the class responsible for translating things like Unix Time to a string, and the View handles how strings look on the screen. A robust View Model might support several views (VM.TimeAsAudio(), VM.TimeASSentence(), etc). A View just wants a string or a number, some data to bind to. You could rewrite the model from a binary tree into an SQL database into a decryption tool and the View wouldn't need to change at all, it only talks to the ViewModel.

My understanding was that the Managers are sort of utility classes that help the Scene Representations stay coordinated/initialized with the device models, and helps the Views (User Abstraction Layer) get references to the right Scene Representations. I could see that described as part of the View Model, or alternatively simply a utility outside the MVVM paradigm (like a DNS service isn't specifically a part of the client-server paradigm, but is a necessary utility for implementing a client-server paradigm).